tests/phpunit/CRM/Financial/BAO/FinancialAccountTest.php

```diff
@@ -177,4 +177,42 @@ public function testGetFinancialAccountByFinancialTypeAndRelationshipCustomAdded
       CRM_Financial_BAO_FinancialAccount::getFinancialAccountForFinancialTypeByRelationship(2, 'Credit/Contra Revenue Account is'));
   }
 
+  /**
+   * Test getting deferred financial type.
+   */
+  public function testGetDeferredFinancialType() {
+    $params = array(
+      'name' => 'TestFinancialAccount_1',
+      'accounting_code' => 4800,
+      'contact_id' => 1,
+      'is_deductible' => 0,
+      'is_active' => 1,
+      'is_reserved' => 0,
+    );
+
+    $ids = array();
+    $financialAccount = CRM_Financial_BAO_FinancialAccount::add($params, $ids);
+    $params['name'] = 'test_financialType1';
+    $financialType = CRM_Financial_BAO_FinancialType::add($params, $ids);
+    $relationTypeId = key(CRM_Core_PseudoConstant::accountOptionValues('account_relationship', NULL, " AND v.name LIKE 'Deferred Revenue Account is' "));
```

Comment:
Once again a nice-to-have but you seem to be using this one-liner often enough I feel it warrants a function on the BAO getAccountRelationshipType('Deferred Revenue Account is') or similar